src/modules/users/Config.cpp

```diff
@@ -23,6 +23,17 @@
 #include <QFile>
 #include <QRegExp>
 
+#ifdef HAVE_ICU
+#include <unicode/translit.h>
+#include <unicode/unistr.h>
+
+//Did my best choosing compound ID
+static const char* TRANSLITERATOR_ID = "Russian-Latin/BGN;"
```

Comment:
`static const char TRANSLITERATOR_ID[] =` (for literals that's idiomatic rather than having the array implicit and declaring the pointer)

Reply:
But if you happen to use the same literal more than once, the pointer version can be merged by the compiler, the array version you suggest cannot.

On the other hand, the pointer version introduces an extra indirection and an extra relocation compared to the array version.